src/pages/invoices/common/components/ClientSelector.tsx

```diff
@@ -106,9 +106,11 @@ export function ClientSelector(props: Props) {
             <div>
               <p className="text-sm text-gray-700">{contact.email}</p>
 
-              <Link to={resource.invitations[0].link} external>
-                {t('view_in_portal')}
-              </Link>
+              {resource.invitations.length && (
```

Comment:
This code smells. I understand it works, but please be specific with >= 1. length property is always there.